Gemfile.rails3

```diff
@@ -0,0 +1,6 @@
+source 'https://rubygems.org'
+
+# Specify your gem's dependencies in hestia.gemspec
+gemspec
+
+gem "actionpack", ">= 3", "< 4"
```

Comment:
I'm going to feel like an idiot, but is there a reason this isn't just "~> 3.0"?